src/ripple/app/consensus/LedgerConsensus.cpp

```diff
@@ -1512,13 +1518,13 @@ class LedgerConsensusImp
     */
     void takeInitialPosition (Ledger& initialLedger)
     {
-        SHAMap::pointer initialSet;
+        std::shared_ptr<SHAMap> initialSet;
```

Comment:
The last thing we need is to add an anonymous nested function into `takeInitialPosition`